src/components/admin/Checkbox/Checkbox.tsx

```diff
@@ -30,21 +29,17 @@ export const Checkbox: React.FC<CheckboxProps> = ({
   subtext,
   variant = "checkbox",
   tabIndex,
-  ml,
   ...inputProps
 }) => {
   const error = get(errors, name);
 
   return (
-    <fieldset
-      className={`Checkbox mb-7 mt-4 ${ml && `ml-${ml}`}`}
-      tabIndex={tabIndex}
-    >
+    <fieldset className="mb-7 mt-4" tabIndex={tabIndex}>
       <div className="relative flex items-start">
         <div className="flex items-center h-5">
           <input
             {...inputProps}
-            className="AdminCheckbox__input checked:bg-blue-500 indeterminate:bg-gray-300 focus:ring-indigo-500 h-4 w-4 text-indigo-600 border-gray-300 rounded"
+            className="checked:bg-blue-500 indeterminate:bg-gray-300 focus:ring-indigo-500 h-4 w-4 text-indigo-600 border-gray-300 rounded"
```

Comment:
[NOTE]
Can also do what @sunny-viktoryia did in earlier PRs and move this string in a new prop at the start of the tag i.e.

```tsx
<input 
  data-bem="AdminCheckbox__input"
  {...inputProps}
```

This way we can still use the browser inspect to quickly find our way through the DOM

Reply:
added, [2d30aa4](https://github.com/sparkletown/sparkle/pull/2930/commits/2d30aa4b14d22ebc719f307bc8fb1b25b21557e5)